Implement Provider content type (#3)
collective/collective.casestudy: README.md

</div>

## Features

> **collective.casestudy** provides two new content types to be used in Plone sites:

Reviewer: All instances of `collective.casestudy` should be changed from bold format to an inline literal. I made just this one suggestion to not repeat myself.

```suggestion
`collective.casestudy` provides two new content types to be used in Plone sites, Case Study and Provider.
```

### Case Study

> A case study of a Plone deployment, which has attributes to track the Plone version used, the industry and the type of usage of Plone.

Reviewer: ```suggestion
A case study of a Plone deployment, which has attributes to track the Plone version used, the industry, and the type of usage of Plone.
```

Start Plone and activate the plugin in the addons control-panel.

## Source Code and Contributions

> If you want to help with the development (improvement, update, bug-fixing, ...) of `collective.casestudy` this is a great idea!

Reviewer: ```suggestion
We welcome contributions to `collective.casestudy`.
You can create an issue in the issue tracker, or contact a maintainer.
```

> We appreciate any contribution and if a release is needed to be done on PyPI, please just contact one of us.

Reviewer: Combined with previous.

```suggestion
```

### Development

> You need a working `python` environment (system, virtualenv, pyenv, etc) version 3.7 or superior.

Reviewer: Python is a proper noun, not an inline literal.

Also check the Python version.

```suggestion
You need a working Python environment version 3.7 or later.
```

```

> To run tests for this package:

Reviewer: ```suggestion
```
